tests/utils/streams/test_dummy.py

```diff
@@ -40,8 +40,6 @@ def get_producer(self) -> DummyProducer[int]:
     def get_payloads(self) -> Iterator[int]:
         return itertools.count()
 
-    @pytest.mark.xfail(
-        strict=True, reason="rebalancing not implemented", raises=NotImplementedError
```

Comment:
This now raises a different error because `DummyBroker` doesn't support rebalancing multiple consumers within a consumer group.